torch/testing/_asserts.py

```diff
@@ -654,6 +681,10 @@ def assert_close(
     Sparse COO tensors are only considered close if both are either coalesced or uncoalesced (if
     :attr:`check_is_coalesced` is ``True``).
 
+    If :attr:`actual` and :attr:`expected` are quantized, they are considered close if they have the same
```

Comment:
can you add dtype as well

Reply:
The `dtype` will always be checked (if `check_dtype` is `True`) regardless of quantization. Thus, we don't mention it here but rather in the first paragraph of the docstring.

Reply:
sg, thanks